utils/spacewalk-common-channels

```diff
@@ -471,7 +471,7 @@ Create everything as well as unlimited activation key for every channel:
         if options.verbose:
             # an empty line after channel group
             sys.stdout.write("\n")
-
+        existing_repo_urls = get_existing_repos(client)
```

Comment:
I do not like this very much. It contacts always SUMA API. When you add 20 repos in one shot, it will call the API again and again. But ok, if you think it is better. Ok with me.

Reply:
I understand your concern and I share it. However, if you add 20 repos in one shot, we are already contacting the API to add them, so at then end I don't think this will make it much worse. If this become and issue, it will be already an issue for creating the repos.

If that was the case, I would vote for a solution that also includes the creation of the repos (i.e. an object that "accumulates" the changes and does one single commit at the end, kinda cache).